awx/sso/backends.py

```diff
@@ -383,31 +378,70 @@ def on_populate_user(sender, **kwargs):
             force_user_update = True
             logger.warning('LDAP user {} has {} > max {} characters'.format(user.username, field, max_len))
 
-    # Update organization membership based on group memberships.
     org_map = getattr(backend.settings, 'ORGANIZATION_MAP', {})
+    team_map = getattr(backend.settings, 'TEAM_MAP', {})
+
+    # Move this junk into save of the settings for performance later, there is no need to do that here
+    #    with maybe the exception of someone defining this in settings before the server is started?
+    # ==============================================================================================================
+    # Get all of the orgs in the DB by name and create any new org defined in LDAP
+
+    # IS THIS ACTUALLY GETTING JUST NAMES OR DOES IT GET ALL AND THEN GET NAMES FORM THEM ALL?
+    existing_org_names = list(Organization.objects.all().values_list('name', flat=True))
```

Comment:
might be able to do `Organization.objects.all().only('name')` though tbh I'm not 100% on difference

Reply:
I checked and this is doing a query specifically for 'name' field:
```
tools_postgres_1    | 2022-09-24 13:41:19.087 UTC [111] LOG:  statement: SELECT "main_organization"."name" FROM "main_organization" ORDER BY "main_organization"."name" ASC
```

Reply:
The only difference is that `.only()` still builds the python model objects with incomplete fields. If you're only going to use the name strings, then I think values_list is preferable.